sdk/spring/azure-identity-spring/src/main/java/com/azure/spring/identity/AzureCloud.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.identity;
+
+
+/**
+ * Enum to define the authority host uri for all Azure Cloud environments: Azure, Azure China...
+ */
+public enum AzureCloud {
```

Comment:
This configuration is available [here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/identity/azure-identity/src/main/java/com/azure/identity/AzureAuthorityHosts.java#L9) in azure-identity SDK, 

Since spring identity SDK takes a compile dependency on azure-identity. 
It can use the `AzureAuthorityHosts` class from it.

Reply:
Thanks, we will refactor our implemention here then